src/ProcessManager.php

```diff
@@ -818,12 +840,13 @@ public function restartSlaves()
         }
 
         $this->inReload = true;
-        $this->output->writeln('Restarting all workers');
+        $start = microtime(true);
 
         $this->closeSlaves();
         $this->createSlaves();
 
         $this->inReload = false;
+        $this->output->writeln(sprintf("Workers have been restarted in %.3f ms.", (microtime(true) - $start) * 1000));
```

Comment:
Since those two methods (closeSlaves and createSlaves) are not blocking and the real restart is not the measured time, it doesn't make much sense to print that.

Reply:
done